superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -19,6 +19,7 @@
 import React, { useState, useEffect } from 'react';
 import { styled } from '@superset-ui/core';
 import { debounce } from 'lodash';
+import { Global, css } from '@emotion/react';
```

Comment:
Let's pull these in from `superset-ui/core` - I know `css` is exported from there, and I _think_ `Global` is too.